duckduckgo/iOS
# Support for basic authentication

Task/Issue URL: https://app.asana.com/0/856498667320406/1133521518040402

**Description**:
Support for basic authentication as requested in #467 

**Steps to test this PR**:
1. Open website requesting basic authentication.
2. Check if authentication is possible.

Note: there are two variants of alert message: one for HTTP and another one for HTTPS connection.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
